lib/receipt.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/**
+ * @module striim-sdk
+ */
+
+/**
+ * @class Receipt
+ * A class for modelling a _hubii striim_ payment receipt.
+ */
```

Comment:
Needs the @alias thingy... see Payment class.

Reply:
Will add.